cumulus_library/studies/core/core_templates/observation.sql.jinja

```diff
@@ -52,15 +51,12 @@ SELECT
     id,
     category_code,
     category_code_system,
-    component,
```

Comment:
the fields removed here were not flattened - i'll swing back around on the required/must support fields on these.